manifest/index.go

```diff
@@ -504,7 +505,15 @@ func (wixFile *WixManifest) Normalize() error {
 		if v.Major() > 255 || v.Minor() > 255 || v.Patch() > 65535 {
 			return fmt.Errorf("Failed to parse version '%v', fields must not exceed maximum values of 255.255.65535", wixFile.Version.User)
 		}
-		wixFile.Version.MSI = v.String()
+
+		if n, err := strconv.ParseInt(v.Metadata(), 10, 64); err == nil {
+			// append a metadata version number if present
+			wixFile.Version.MSI = fmt.Sprintf("%d.%d.%d.%d", v.Major(), v.Minor(), v.Patch(), n)
+		} else {
+			// only use major, minor, and patch if no numeric metadata
+			wixFile.Version.MSI = fmt.Sprintf("%d.%d.%d", v.Major(), v.Minor(), v.Patch())
+		}
+
 		wixFile.Version.Hex = v.Major()<<24 + v.Minor()<<16 + v.Patch()
```

Comment:
I think we'll want to account for the n if specified here, e.g.
```suggestion
		wixFile.Version.Hex = v.Major()<<32 + v.Minor()<<24 + v.Patch() << 8 + n
```

(this would allow `n` here to be up to 255, which is probably enough for us)